spec/rspec/mocks/verifying_doubles/class_double_with_class_loaded_spec.rb

```diff
@@ -28,6 +28,12 @@ module Mocks
         prevents { expect(o).to receive(:defined_instance_method) }
       end
 
+      it "suggests using `instance_double` when an instance method is stubbed" do
+        o = class_double("LoadedClass")
+        prevents(/Perhaps you meant to use `instance_double`/) { allow(o).to receive(:defined_instance_method) }
+      end
```

Comment:
I think we also need a negative test -- that is, a test of a method that is not an instance method, to assert that it doesn't recommend using `instance_double`.  As the specs currently exist, we could remove the `if InstanceMethodReference.for(doubled_module, method_name).implemented?` conditional and simply _always_ append the "Perhaps..." bit, and all the specs would pass.